src/test/java/com/github/havardh/javaflow/JavaFlowTest.java

```diff
@@ -191,7 +191,9 @@ public void shouldMapString() {
         public void shouldMapCharArrayToString() {
```

Comment:
We should make this two distinct tests, as the original mapped a `char[]` into a string type, while the added cases operates on arrays :) So lets add `shouldMapGenericArrays`. We could also rename the old test model to `ModelWithCharArray`.

Reply:
Fixed!